### PR TITLE
Clarify zero address usage in proposer and executor roles in Timelock

### PR DESCRIPTION
**Pull request by spalladino**
This is not documented in the API docs, only in the [Access Control article](https://docs.openzeppelin.com/contracts/4.x/access-control).

### CODE REVIEW

File: contracts/governance/README.adoc [hunk 1]
```diff
@@ -64,6 +65,8 @@ Operations status can be queried using the functions:
 [[timelock-roles]]
 ==== Roles
 
+Proposer and Executor roles can be granted to the zero address to open them up and allow anyone to act as such a role.
```
Comment (Amxx) [inline contracts/governance/README.adoc:68]: Its only true of executor. We removed that feature from Proposer for security reasons (proposers are also cancelors)

**Comment by spalladino** [comment]
We'll need to update the article then:

> There is an additional feature built on top of AccessControl: giving the
proposer or executor role to address(0) opens access to anyone.

**Comment by spalladino** [comment]
@Amxx updated!

File: docs/modules/ROOT/pages/access-control.adoc [hunk 2]
```diff
@@ -198,7 +198,7 @@ The first step in configuring the xref:api:governance.adoc#TimelockController[`T
 
 Roles are managed using the xref:api:access.adoc#AccessControl[`AccessControl`] interface and the `bytes32` values for each role are accessible through the `ADMIN_ROLE`, `PROPOSER_ROLE` and `EXECUTOR_ROLE` constants.
 
-There is an additional feature built on top of `AccessControl`: giving the proposer or executor role to `address(0)` opens access to anyone. This feature, while potentially useful for testing and in some cases for the executor role, is dangerous and should be used with caution.
+There is an additional feature built on top of `AccessControl`: giving the proposer role to `address(0)` opens access to anyone to execute a proposal once the timelock has expired. This feature, while useful, should be used with caution.
```
Comment (Amxx) [inline docs/modules/ROOT/pages/access-control.adoc:201]:
```
giving the proposer role to `address(0)`
```
should be 
```
giving the executor role to `address(0)`
```

Comment (spalladino) [reply docs/modules/ROOT/pages/access-control.adoc:201]: Damn. Thanks!

**Review by Amxx — APPROVED** [review]

**Pull request merged**